src/modules/pageManager.js

```diff
@@ -12,5 +13,46 @@ export default class PageManager {
     });
   }
 
+  pullScoresFromAPI = () => {
+    const allScores = Scores.getAPI();
+    allScores.then((scores) => {
+      Scores.scores = [];
+      this.scoreContainer.innerHTML = '';
+      scores.forEach((score) => {
+        console.log(`${score.user}: ${score.score}`);
```

Comment:
- [ ] Please, kindly delete this debugging `console.log` statement from the repo, according to the Javascript required best practices.